### DISCUSSION TITLE
HAVE BEEN ASKING GITHUB SINCE 2022 WHY OUR DEV ACCOUNT HAS BEEN SUSPENDED, NO RESPONSE?!

### DISCUSSION THREAD

**Discussion by KevinHelton** [community/community#60862] [Repositories]
### Select Topic Area

Question

### Body

I Start by saying that I am not a coder.
I am the owner of a project that has used github sind 2011.

In 2022 after supporting github for over a decade i cannot access the account.
All i get is the below message—

Access to your account has been suspended due to a violation of our [Terms of Service]

I have contacted support on numerous instances to find out what has happend and how to rectify the problem.

Its extremely frustrating and unprofessional that they will not respond.

What does it take to get a response from these guys?!!

Please help

**Comment by queenofcorgis** [comment]
Hi @KevinHelton,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your posts were closed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
